### ISSUE TITLE
do we need page-related metadata in the non-visual reading book?

### ISSUE THREAD

**Issue by marisademeglio** [daisy/epub-accessibility-tests#90]
Ace reports that there should be a pagelist when the accessibility metadata includes `pageBreakMarkers`. The non-visual reading book has this metadata but does not have a pagelist. I think we should remove `pageBreakMarkers`. 

Also the accessibility metadata includes `pageNavigation`, which does not raise an Ace violation, but I don't see how it makes sense in this publication. None of the techniques described in [EPUB Accessibility 1.1 Page Navigation](https://www.w3.org/TR/epub-a11y-tech-11/#sec-epub-page-navigation) would apply here as there are no static page markers anywhere in the book.

cc @GeorgeKerscher

**Comment by marisademeglio** [comment]
Page metadata also removed here
https://github.com/daisy/epub-accessibility-tests/pull/92
